### ISSUE TITLE
Update URL for starting a travel claim

### ISSUE THREAD

**Issue by wullaski** [department-of-veterans-affairs/va.gov-team#97545]
- Update the link to start a travel claim

```
        href={`/appointments/claims/?date=${appointment.start}`}
```
Update to:
href={`/appointments/claims/?id=${appointment.id}`}

- Make sure response objects match mocks

more info in this [thread](https://dsva.slack.com/archives/C02G6AB3ZRS/p1738270254540209)

**Comment by loripusey** [comment]
@wullaski find out if we know what the URL is and if we do finish this ticket; if we don't put it back on the backlog

**Comment by wullaski** [comment]
@simiadebowale This is the link from the appointment details view that starts the SMOC flow. Do we know what the URL for this is going to be?

**Comment by loripusey** [comment]
@wullaski do we also need to update the URL for checking you claim status (i.e. the under-development claim status tool) for past appts for which you already submitted a claim?

**Comment by wullaski** [comment]
I don't think so. That one was good AFAIK and was updated a few weeks ago by @liztownd as part of wider link [URL change PR](https://github.com/department-of-veterans-affairs/vets-website/pull/33498).

**Comment by wullaski** [comment]
@loripusey this one is ready to be pulled in to the next sprint

**Comment by wullaski** [comment]
@loripusey this has been merged in
